.github/workflows/ci-cd.yml

```diff
@@ -20,11 +20,38 @@ jobs:
         check: ['tests', 'pre-commit', 'lint']
         python-version: ["3.10.12"]
         poetry-version: ["1.8.1"]
+    services:
+      postgres:
+        image: postgres:15-alpine
+        env:
+          POSTGRES_USER: sbomify
+          POSTGRES_PASSWORD: sbomify
+          POSTGRES_DB: sbomify
```

Comment:
Shouldn't we just change this to `sbomify_tests` and then we can drop :129 and :130?

Reply:
Django test runner creates the test DB and needs permission to do that. The DB in this case would be `sbomify_tests`. So it's better to let it do it so the tests don't break.